recipes/colormath/meta.yaml

```diff
@@ -0,0 +1,47 @@
+
```

Comment:
As in my other PR, GitHub is being confusing about which line this comment is on. Head of file or between `{% set` and `package`?